# Type of the devices required
string type

---

# List of components as yaml string
temoto_component_manager/Component[] component_infos
